README.md

```diff
@@ -1,13 +1,123 @@
+
+To build the sidecar, check out the sidecar project:
+
+```
+mkdir -p $GOPATH/src/github.com/SUSE
+git clone https://github.com/SUSE/cf-usb-sidecar $GOPATH/src/github.com/SUSE/cf-usb-sidecar
+cd $GOPATH/src/github.com/SUSE/cf-usb-sidecar
+```
+
+Configure your Docker repository information:
+
+```
+export DOCKER_REPOSITORY=docker.io
+export DOCKER_ORGANIZATION=splatform
+
+docker login # if authorization is required
+```
+
+Then build the top level dependencies and the sidecar:
+
+```
+make tools
+make build-image
+cd csm-extensions/services/dev-mysql
+make build-image public-image helm
+```
+
+The generated Helm chart will be available in the `chart/` directory.
+
+Install the Helm chart:
+
+```
+# You will need to know the namespaces and domain for your cluster:
+UAA_NAMESPACE=uaa
+CF_NAMESPACE=cf
+CF_DOMAIN=cf-dev.io
+
+SIDECAR_NAMESPACE=mysql
+
+UAA_CA_CERT="$(kubectl get secret secret --namespace ${UAA_NAMESPACE} -o jsonpath="{.data['internal-ca-cert']}" | base64 --decode -)"
+CF_CA_CERT="$(kubectl get secret secret --namespace ${CF_NAMESPACE} -o jsonpath="{.data['internal-ca-cert']}" | base64 --decode -)"
+CF_PASSWORD="$(kubectl get secret secret --namespace ${CF_NAMESPACE} -o jsonpath="{.data['cluster-admin-password']}" | base64 --decode -)"
+
+helm install ./chart --name mysql-instance --namespace ${SIDECAR_NAMESPACE} \
+	--set "env.UAA_CA_CERT=${UAA_CA_CERT}" \
+	--set "env.CF_CA_CERT=${CF_CA_CERT}" \
+	--set "env.SERVICE_LOCATION=http://cf-usb-sidecar-mysql.${SIDECAR_NAMESPACE}.svc.cluster.local:8081" \
+	--set "env.SERVICE_MYSQL_HOST=AUTO" \
+	--set "env.CF_ADMIN_USER=admin" \
+	--set "env.CF_ADMIN_PASSWORD=${CF_PASSWORD}" \
+	--set "env.CF_DOMAIN=${CF_DOMAIN}"
+```
+
+Eventually you should see two pods start in the `SIDECAR_NAMESPACE`:
+
+```
+$ kubectl get pods --namespace ${SIDECAR_NAMESPACE}
+NAME                                      READY     STATUS    RESTARTS   AGE
+cf-usb-sidecar-mysql-b44d4d66f-d27qb      1/1       Running   0          2m
+mysql-0                                   1/1       Running   0          2m
+```
+
+Once the pods are ready, it should be available in the marketplace:
+
+```
+$ cf marketplace
+Getting services from marketplace in org org / space space as admin...
+OK
+
+service    plans     description
+postgres   default   Default service
+mysql      default   Default service
+
+TIP:  Use 'cf marketplace -s SERVICE' to view descriptions of individual plans of a given service.
+```
+
+At this point, services can be made available to apps. In this case we're going to use the [django-cms](https://github.com/scf-samples/django-cms) app.
+
+```
+git clone https://github.com/scf-samples/django-cms -b scf
+cd django-cms
+cf create-service postgres default django-cms-db
+cf push --no-start django-cms
+cf set-env django-cms DISABLE_COLLECTSTATIC 1
+cf set-env django-cms DJANGO_SETTINGS_MODULE settings
+cf restart django-cms
```

Comment:
`cf start` is good enough, given the `--no-start` argument to `push`.

Reply:
Fixed in d2eace9